src/components/ContentOpenWith/__tests__/ExecuteForm-test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ExecuteForm from '../ExecuteForm';
+
+describe('components/ContentOpenWith/ExecuteForm', () => {
+    const executePostData = {
+        url: 'foo.com',
+        params: [{ key: 'foo', value: 'bar' }, { key: 'bar', value: 'baz' }],
+    };
+
+    const submitStub = jest.fn();
+    const ref = {
+        current: {
+            submit: submitStub,
+        },
+    };
+    const onSubmitStub = jest.fn();
+
+    const getWrapper = props => shallow(<ExecuteForm {...props} />);
+
+    describe('componentDidMount', () => {
+        it('should submit the form and call callback on mount', () => {
+            React.createRef = jest.fn().mockReturnValue(ref);
```

Comment:
Consider moving the mocking of this method to a top-level `React` module, similar to `build/react-intl-mocks.js`.